api/client/exec.go

```diff
@@ -17,6 +17,7 @@ import (
 // Usage: docker exec [OPTIONS] CONTAINER COMMAND [ARG...]
 func (cli *DockerCli) CmdExec(args ...string) error {
 	cmd := Cli.Subcmd("exec", []string{"CONTAINER COMMAND [ARG...]"}, Cli.DockerCommands["exec"].Description, true)
+	escapeKeys := cmd.String([]string{"-escapeKeys"}, "", "Set up escape key sequence")
```

Comment:
i think it goes hand in hand... and yeah i agree it's going to be painful... such a mistake :( Anyway, you don't need to change it in this PR